doc/data/messages/c/consider-using-namedtuple-or-dataclass/bad.py

```diff
@@ -0,0 +1,5 @@
+# Subdicts have at least 1 common key
+MAPPING_1 = {  # [consider-using-namedtuple-or-dataclass]
```

Comment:
Could you make the example be about an object or animal instead ? A cat with attribute paw / fur for example.

Reply:
Changed